fastlane/actions/xchtmlreport.rb

```diff
@@ -0,0 +1,118 @@
+module Fastlane
```

Comment:
#? is this going to be rolled into a plugin at some point, so we don't need to include it ourselves?

Reply:
Yep that's the plan! It depends on my [JUnit PR](https://github.com/TitouanVanBelle/XCTestHTMLReport/pull/63) and a new stable release of xchtmlreport before I can upstream the fastlane plugin.